src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
 // write your code here
+
+const rows = document.querySelectorAll('tr');
+
+rows.forEach((row) => {
+  const secondChild = row.children[1];
+  const lastChild = row.children[row.children.length - 1];
+  let copySecondChild = document.createElement('td');
```

Comment:
Issue: The variable 'copySecondChild' is being reassigned a new element type ('th') inside the if block. Consider declaring 'copySecondChild' with 'let' instead of 'const' to allow reassignment.